data/index.js

```diff
@@ -69,7 +69,32 @@ export function registerReducer( reducerKey, reducer ) {
  *                              state as first argument.
  */
 export function registerSelectors( reducerKey, newSelectors ) {
-	selectors[ reducerKey ] = newSelectors;
+	const store = stores[ reducerKey ];
+	const createStateSelector = ( selector ) => ( ...args ) => selector( store.getState(), ...args );
+	selectors[ reducerKey ] = mapValues( newSelectors, createStateSelector );
```

Comment:
Initially, I wanted to avoid having to create a wrapper for each selector but I do see the DevX benefits from this new API